typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -22,6 +22,7 @@ export enum ExplorerFamily {
   Etherscan = 'etherscan',
   Blockscout = 'blockscout',
   Routescan = 'routescan',
+  zksync = 'zksync',
```

Comment:
nit: matching the tech stack enum
```suggestion
  ZKSync = 'zksync',
```